app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    post.comments_counter = post.comments.count
+    post.save
+  end
 end
```

Comment:
- [ ]  You have correctly added the `update_comments_counter` method as required. However, a more efficient way to achieve this is using the `after_save` callback. Additionally, the comments counter should be updated using the `increment` method, and it's a good practice to make this method private.

  Suggested Changes:
  
  ```ruby
  class Comment < ApplicationRecord
    belongs_to :user
    belongs_to :post
  
    after_save :update_comments_counter
  
    private
  
    def update_comments_counter
      post.increment!(:comments_counter)
    end
  end
  ```